src/containers/CardList.js

```diff
@@ -13,7 +13,7 @@ class CardList extends PureComponent {
     };
   }
 
-  handleClick = (func, e) => {
```

Comment:
You have to bind `hadleClick` to the component  in `constructor` to make it work:
```javascript
this.handleClick = this.handleClick.bind(this);
```
You can also remove it completely since it's not doing anything right now. I kept it in case we want to show result for only active card.

Reply:
I just added the binding, when I removed it there were errors when pressing the run button.